### ISSUE TITLE
"Current" step status in the timeline is announced twice (current current)

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4603]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.1.1 Non-text Content

**Test Environment:**
- Test URL: https://clarity.design/documentation/timelines
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Select any one of the tabs provided for horizontal, vertical and angular timelines.
3. Move focus on the current state step and observe the screen reader&#039;s spoken feedback.

**Expected result:**
Current state should be announced appropriately as "current"

**Actual result:**
Current state is announced as "current, current"

**Root cause:**
Aria-label is set to "current" and Aria-current="true" attribute is also used.

**Reporter Email:** gkhadke@vmware.com

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
